CloudyCider/Views/EC2/EC2List.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  EC2List.swift
+//  CloudyCider
+//
+//  Created by Matvii Hodovaniuk on 7/7/19.
+//  Copyright © 2019 Matvii Hodovaniuk. All rights reserved.
+//
+
+import SwiftUI
+
+extension View {
+  /// Returns a type-erased version of the view.
+  public var typeErased: AnyView { AnyView(self) }
+}
```

Comment:
since this will be used not only in `EC2List`, it should be moved outside of this file